### PR TITLE
refactor(multidropzone): [EMU-5671] Add file maxSize validation support

### PR DESCRIPTION
**Pull request by diogomateus**
### What this PR does
Add file maxSize validation support to MultiDropZone component.

_Please include a summary of the change(s)._
1. Added support to maxSize prop and internal validation of size
2. Added formatBytes util to show more human readable information on file size

Solves:  
EMU-5671

Screenshot 2023-01-27 at 10 45 04

### Checklist:

- [ ] I reviewed my own code
- [ ] The changes align with the designs I received  
  Or give a reason why this does not apply:
- [ ] I have attached screenshot(s), video(s) or gif(s) showing that the solution is working as expected  
  Or give a reason why this does not apply:
- [ ] I have updated the task(s) status on Linear
- [ ] All new media is optimized (images, gifs, videos)

### Browser support

My code works in the following browsers:

- [ ] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/lib/util/formatBytes/index.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const k = 1024;
+const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+
+export function formatBytes(bytes: number, decimals = 0): string {
```
Comment (diogomateus) [inline src/lib/util/formatBytes/index.ts:4]: Based on this stack overflow answer:
https://stackoverflow.com/questions/15900485/correct-way-to-convert-size-in-bytes-to-kb-mb-gb-in-javascript

Which is based on this script here, but was updated to be ES6:
https://web.archive.org/web/20120507054320/http://codeaid.net/javascript/convert-size-in-bytes-to-human-readable-format-(javascript)

File: src/lib/components/multiDropzone/index.test.tsx [hunk 2]
```diff
@@ -60,6 +60,21 @@ describe('MultiDropzone component', () => {
...
     });
 
+    it("should show max file size error message", async () => {
+      const screen = setup({maxSize: 10 });
+      const input = screen.getByTestId(inputTestId);
+      const bigFile = file;
+      Object.defineProperty(bigFile, 'size', { value: 1024 });
+
+      await act(async () => {
+        fireEvent.change(input, { target: { files: [bigFile] } });
```
Comment (talo242) [inline src/lib/components/multiDropzone/index.test.tsx:70]: _This comment does not require any changes in this PR_
I think we'll have to update `testing-library` and add `user-event` pretty soon. As noted in the testing-library documentation, [using fireEvent is not recommended and instead we should use user-event](https://testing-library.com/docs/user-event/intro#differences-from-fireevent).
Can you create a follow up ticket for tackling this in the future?

**Review by talo242 — APPROVED** [review]
Left a comment regarding a follow up ticket, but from my point of view this PR is ready to be merged :shipit:

File: src/lib/components/multiDropzone/index.test.tsx (hunk 2 above)
Comment (diogomateus) [reply src/lib/components/multiDropzone/index.test.tsx:70]: Noted and 100% agree!

**Pull request merged**